ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -133,12 +143,26 @@ func convertToComposeService(containerDefinition *ecs.ContainerDefinition) (comp
 		Logging:     logging,
 		Volumes:     volumes,
 		Ports:       ports,
+		Networks:    networks,
 		Sysctls:     sysctls,
 	}
 
 	return service, nil
 }
 
+func convertToNetworks() map[string]*composeV3.ServiceNetworkConfig {
+	out := make(map[string]*composeV3.ServiceNetworkConfig)
+	aliases := []string{}
+	aliases = append(aliases, network.EcsLocalNetworkName)
+	out[network.EcsLocalNetworkName] = &composeV3.ServiceNetworkConfig{
```

Comment:
It looks like [aliases](https://docs.docker.com/compose/compose-file/#aliases) are used to give a container an alternative name in the same network. So I don't think we want to actually use this!

If we want to get:
```yaml
networks:
- ecs-local-network
```
Instead, I think we can give a value of `nil`. Can you try this and get rid of the function:
```go
Networks: map[string]*composeV3.ServiceNetworkConfig{
     network.EcsLocalNetworkName: nil
}
```

Reply:
Yeah you are right! I changed and manually tested it.